src/integration-test/resources/LetterIT/__files/test2_ErrorFromDigital_SuccessfulSnailMail/mappings/snailmail-sender-batch.json

```diff
@@ -1,12 +1,12 @@
 {
   "request": {
-    "method": "POST",
-    "url": "/snailmail-sender/send/snailmail"
+    "urlPathPattern" : "/snailmail-sender/send/batch/.*",
+    "method": "POST"
   },
   "response": {
     "headers": {
       "Content-Type": "application/json"
     },
-    "status": 200
+    "status": 201
```

Comment:
According to API (also in this PR) HTTP 200 is the status code for successful calls.